.env.example

```
@@ -22,10 +22,47 @@ DB_PASSWORD=secret
+MEMCACHED_HOST=127.0.0.1
 MEMCACHED_PORT=11211
+
+# Redis (optional, if using redis cache, for Docker use container name 'redis', for local use 'localhost' or '127.0.0.1')
+#REDIS_HOST=127.0.0.1
+#REDIS_PORT=6379
+#REDIS_PASSWORD=
+#REDIS_DB=0
+
+# Mail settings
+MAIL_SMTP_ENABLED=false
+MAIL_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAIL_HOST key should go before the MAIL_SMTP_ENABLED key

(UnorderedKey)